layout/templates/key_stories.mustache

```diff
@@ -1,54 +1,62 @@
-<div id="key-stories" class="viz-text">
+<div id="wu-story" class="viz-text">
 
-  <h3>How do we use water in the U.S.?</h3>
+  <div id="main-story">
+  
+    <h3>How do we use water in the U.S.?</h3>
   
     <div class="key-story-plot" id="national-pie">
-    <p>
-    The U.S. states and territories used X billion gallons per day in 2015. This amount of water is approximately equal to YY (Some percentage of Lake Erie?). We use water in different ways to serve many economically and culturally important needs. The breakdown of different water uses is as follows:
-    </p>
+      <p>
+      The U.S. states and territories used over 322 billion gallons per day in 2015. This amount of water is approximately equal to filling the Empire State Building over 1000 times per day. We use water in different ways to serve many economically and culturally important needs. The national breakdown is as follows:
```

Comment:
`Approximately...over 1000 `
I think you can skip approximately, since you are saying "more than"